mpas_analysis/shared/plot/plotting.py

```diff
@@ -761,7 +761,10 @@ def plot_vertical_section(
     ylabel=None,
     fileout='moc.png',
     figsize=(10, 4),
-    dpi=300):  # {{{
+    dpi=300,
+    xLim=None,
+    yLim=None,
+    invertYAxis= True):  # {{{
```

Comment:
Yes, that was my reasoning.  I agree we need to test MOC, in fact the whole MPAS-Analysis.